canonical-web-and-design/maas-ui
# Build set zone form in action menu dropdown.

## Done
- Added a couple of components:
  - `SetZoneForm` - self-explanatory, just a FormikForm with a single zone select
  - `ActionForm` - generic component that will house all the "fieldless" forms, as per canonical-web-and-design/MAAS-squad#1783
- Added `bordered` and `onCancel` props to `FormCardButtons`, and amended the styling a little bit to accommodate.
- It's pretty crude at the moment - there's no progress spinner or error handling, but I'll implement them as part of #881 once I've built a few more of these forms and got a feel for how they should be shaped.

## QA
- Select some machines
- Click the "Take action" menu and select some actions and check that they still look fine and display the right text (ignore the green delete button for now)
- Choose "Set zone" from the dropdown and check that you can set the zone of multiple machines at once. Note that progress and errors will be handled in #881.

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1788
